src/lsp/sql_parser/lexer.mll

```diff
@@ -118,18 +124,14 @@ rule token = parse
   | ':' (['a'-'z' 'A'-'Z']['A'-'Z' 'a'-'z' '0'-'9' '_' '-']* as s)
       { COBOL_VAR s }
   | '\\' (['a'-'z' 'A'-'Z']['A'-'Z' 'a'-'z' '0'-'9' '_']* as s)
-      { BACKSLASHED_VAR s }
+      { BACKSLASH_VAR s }
   | ['a'-'z' 'A'-'Z' '0'-'9']['A'-'Z' 'a'-'z' '0'-'9' '-' '_' '*' ]* as s 
-  (*If you want to make a better sql parsing, this is to rework*)
       { get_keyword s }
-  | ['0'-'9']+ as s
-      { DIGITS s }
+  | numbers as n
+      { DIGITS n }
```

Comment:
So here it's probably `INTEGER` instead of `DIGITS`.

Reply:
I change by number, because it can be a FLOAT